content/_meta/info/alert.md

```diff
@@ -2,6 +2,7 @@
 live: true
 info_slug: alert
 title: alert
+aria: BLM
```

Comment:
This approach makes sense. Maybe best to spell it out rather than abbreviating?

Reply:
I need to work out how to stop it from only taking the first word in that case, will do :)